pkg/sfu/downtrack.go

```diff
@@ -99,8 +105,10 @@ type DownTrack struct {
 	closeOnce               sync.Once
 
 	// Report helpers
-	octetCount   atomicUint32
-	packetCount  atomicUint32
+	primaryStats atomic.Value
```

Comment:
better to add a comment about type of Value stored

Reply:
Will do.